src/utils.js

```diff
@@ -278,12 +278,16 @@ export function orderProperties(properties, order) {
 }
 
 export function isMultiSelect(schema) {
-  return Array.isArray(schema.items.enum) && schema.uniqueItems;
+  return schema.items
+    ? Array.isArray(schema.items.enum) && schema.uniqueItems
+    : false;
```

Comment:
Should work just fine with `return schema.items && Array.isArray(schema.items.enum) && schema.uniqueItems`